### PR TITLE
Add display_name field to ClusterUser table in database

### PR DESCRIPTION
**Pull request by Rizwana777**
#### Description:
-  Added new field in cluster user table called display_name
- Removed `internalGetOrCreateClusterUserByNamespaceUID`  function as it implements same logic as `internalProcessMessage_GetOrCreateClusterUserByNamespaceUID`
- unit test

#### Link to JIRA Story (if applicable):
https://issues.redhat.com/browse/GITOPSRVCE-600

### CODE REVIEW

File: backend/eventloop/shared_resource_loop/sharedresourceloop.go [hunk 1]
```diff
@@ -530,7 +530,7 @@ func internalProcessMessage_GetOrCreateClusterUserByNamespaceUID(ctx context.Con
 	if err != nil {
 		if db.IsResultNotFoundError(err) {
 			isNewUser = true
-
+			clusterUser.Display_name = string(workspaceNamespace.Name)
```
Comment (chetan-rns) [inline backend/eventloop/shared_resource_loop/sharedresourceloop.go:533]: `workspaceNamespace.Name` is already of type string. I don't think we need to explicitly convert it again.

**Review by chetan-rns — COMMENTED** [review]
Thanks, @Rizwana777. I've added a few Nit comments everything else looks good to me.

**Comment by Rizwana777** [comment]
@chetan-rns Thanks for the review, Updated the chages

**Review by chetan-rns — APPROVED** [review]
Thanks  @Rizwana777 

/lgtm

File: backend-shared/db/clusteruser.go [hunk 2]
```diff
@@ -197,6 +198,35 @@ func (dbq *PostgreSQLDatabaseQueries) GetClusterUserBatch(ctx context.Context, c
...
+	}
+
+	if result.RowsAffected() != 1 {
+		return fmt.Errorf("unexpected number of rows affected: %d", result.RowsAffected())
+	}
+
+	return nil
+
+}
+
```
Comment (jgwest) [inline backend-shared/db/clusteruser.go:229]: It seems like `UpdateClusterUser` isn't covered by unit tests, it couldn't hurt to add a quick test for this to improve coverage.

**Review by jgwest — CHANGES_REQUESTED** [review]
Looks great @Rizwana777, and thanks @chetan-rns for reviewing! From me, just a quick test request then good to go.

**Review by jgwest — APPROVED** [review]
Looks great, thanks again @Rizwana777, and thanks @chetan-rns for reviewing!

**Pull request merged**